### DISCUSSION TITLE
Intune policy

### DISCUSSION THREAD

**Discussion by Sunyix11** [Micke-K/IntuneManagement#88] [General]
Hi,
by default, the users have no access to write or create fie in root C:\ drive, they are only have a create folder permission. But we have some users who migrated their laptops from "old word" to Intune and they can write and create files in root C:. Is there any Intune policy for setting the ACL and prevent the write access for root C:\ ? Or we need to use the proactive remediation for it? Thank you in advance!

**Comment by Micke-K** [comment]
Hello!

I don't know any native policy that can be used for setting file permissions.

There are multiple ways to do it with script though Proactive Remediations, PowerShell script or Win32 App (with a PowerShell script or whatever cmd line tool you want to use to specify the permissions)

Cheers!
